index.html

```diff
@@ -614,6 +655,142 @@ <h4>
         </section>
       </section>
     </section>
+    <section>
+      <h2>
+        Automation
+      </h2>
+      <p>
+        The Device Posture API pose a challenge to test authors, as fully
+        exercising interface requires physical hardware devices. To address this
+        challenge this document defines a [[WEBDRIVER2]] [=extension commands=]
+        that allows users to control the reported device posture and simulate a
+        real device.
+      </p>
+      <h3>
+        Extension Commands
+      </h3>
+      <h4>
+        Set device posture
+      </h4>
+      <table class="simple">
+        <tr>
+          <th>
+            HTTP Method
+          </th>
+          <th>
+            [=extension command URI Template|URI Template=]
+          </th>
+        </tr>
+        <tr>
+          <td>
+            POST
+          </td>
+          <td>
+            /session/{session id}/deviceposture
+          </td>
+        </tr>
+      </table>
+      <p>
+        This [=extension commands|extension command=] changes device posture to a specific
```

Comment:
```suggestion
        This [=extension command=] changes device posture to a specific
```

Respec handles singular and plural nouns automatically.

Reply:
Fixed in https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/70bba088084478e20882f7773dbd275b3e5cc891